frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx

```diff
@@ -99,34 +182,10 @@ class AddSubCategory extends React.Component<
             <Row>
               <Col md={12}>
                 <Spin tip="Loading..." spinning={this.state.isLoading}>
-                  <Form size="large" onFinish={this.onFinish}>
-                    <Title level={3}>Category</Title>
-                    <Text>Select a category</Text>
-                    <Form.Item name="category" className={mainStyles.formItem}>
-                      <Select
-                        className={mainStyles.formSelect}
-                        placeholder="Please select a category"
-                      >
-                        {this.state.categories.map((category: Category) => {
-                          return (
-                            <Option key={category.id} value={category.id}>
-                              {category.name}
-                            </Option>
-                          );
-                        })}
-                      </Select>
-                    </Form.Item>
-                    <Title level={3}>Name</Title>
-                    <Text>Name of the subcategory</Text>
-                    <Form.Item name="name" className={mainStyles.formItem}>
-                      <Input placeholder="ex: Local Curriculum" />
-                    </Form.Item>
-                    <Form.Item>
-                      <Button type="primary" htmlType="submit">
-                        Add Subcategory
-                      </Button>
-                    </Form.Item>
-                  </Form>
+                  {this.state.subcategory != null &&
+                    this.subCategoryForm('Edit Subcategory')}
+                  {this.state.subcategory == null &&
+                    this.subCategoryForm('Add Subcategory')}
```

Comment:
I think it's better if you could use a generic string to the button.

What about "Save" ?

Reply:
great.